webdriver/tests/support/helpers.py

```diff
@@ -238,6 +238,17 @@ def available_screen_size(session):
         ];
         """))
 
+
+def viewport_size(session):
```

Comment:
For the screenshot tests we already have a helper that is called `viewport_dimensions`. It would be good to have the same name globally and maybe make it a fixture instead so that special variants for screenshots can override them.